torch/distributed/rpc.py

```diff
@@ -73,13 +76,18 @@ def init_rpc(name, backend='pg'):
     use ``pg`` (ProcessGroup) backend, ``init_process_group`` must be invoked
     prior to this method.
 
+    Also to be noted: RPC package only support Python3
+
     Arguments:
         name (str): a globally unique name of the local RPC agent. (e.g.,
                     ``Trainer3``, ``ParameterServer2``, ``Master``, ``Worker1``)
         backend (str): type of RPC backend implementation. Currently,
                        process group backend ``"pg"`` is the only available
                        backend implementation. (default: ``"pg"``).
     """
+    if sys.version_info < (3, 0):
```

Comment:
nit: I saw `test_rpc.py` uses six. Shall we stay consistent in RPC-related files?

@ezyang do we have any preference on `six.PY2` vs `sys.version_info`?

Reply:
PyTorch does NOT have a dependency on six in the main library (we have a shitty version of six in `torch._six`).  @soumith do you remember why we don't directly depend on six? It seems like a pretty benign dependency, as far as dependencies go.